src/registrar/templates/application_organization.html

```diff
@@ -1,6 +1,8 @@
 <!-- Test page -->
 {% extends 'application_form.html' %}
 {% load widget_tweaks %}
+{% load debugger_tags %}
```

Comment:
**question (non-blocking)**: Is there anything in these debugger tags that would make us not even want to include the code into this template in production? I think there is nothing bad that can happen if we don't use the tags, but I thought I would ask.

Reply:
Good catch here as well. I don't think there is any harm, but there's also no point in loading more tags than we need.